.drone.star

```diff
@@ -1255,7 +1255,7 @@ def setupGraphapiOIdC():
 			'php occ a:e graphapi',
 			'php occ a:e openidconnect',
 			'php occ config:system:set trusted_domains 2 --value=web',
-			'php occ config:system:set openid-connect provider-url --value="https://konnectd:9130"',
+			'php occ config:system:set openid-connect provider-url --value="https://idp:9130"',
```

Comment:
@phil-davis is it correct to just change the alias? Or is that stored somewhere magically, so that it has to stay `konnectd`?

Reply:
The name of the pipeline step should be the thing that translated to the address of the docker in which the processes of that step are running. So if the pipeline step is now called `idp` then it should be accessible there.

Reply:
cool, thanks 👍